### ISSUE TITLE
getting llava output in chinese

### ISSUE THREAD

**Issue by YashKumarVerma** [ollama/ollama#1567]
I've tried running in CLI and via network request, but getting reply in chinese mandarin. 

Has someone faced this?
<img width="1376" alt="image" src="https://github.com/jmorganca/ollama/assets/14032427/bd80327a-e42e-45c2-9ad0-4222b6221bb7">
<img width="1123" alt="image" src="https://github.com/jmorganca/ollama/assets/14032427/af4b98bb-11f5-4ced-8d75-9f4ade11ee26">

**Comment by YumcoderCom** [comment]
when I prompt "Describe the image in detail. /Users/..../figure-8-2.jpg" in the cmd everything works as expected. 
But, when running the prompt from langchain (3.9 and 3.10, 3.11) the result was printed in China and was not related to the provided image.

the code :

```
from langchain.prompts import PromptTemplate

model_img = ChatOllama(model="llava:13b")
model_img.invoke("Describe the image in detail. /Users/..../figure-8-2.jpg")
```

AIMessage(content='在这张图片中，你可以看到一个室内场景，有一位女子正在拿着一个小男孩的手。他们似乎在一起参观一个博物馆或者是游览一个室内展览。\n\n场景中还有两位其他人，一人位于女子和小男孩的左侧，另一人则在他们的右侧。这些人都看起来像是游客或者是博物馆参观者。\n\n此外，还有一个手提包在场景中的下方，可能是女子或其他游客的物品。', additional_kwargs={}, example=False)


test with ollama 0.1.{14,15,16}

**Comment by ndjstn** [comment]
I am having a similar issue, not only do I gt Chinese,  it is confused.


```
抱歉，我无法识别该文件的内容。请提供更多信息或上传其他图片以便我可以为您提供有关该图像的分析和解释。

total duration:       12.775150416s
load duration:        8.967403808s
prompt eval count:    31 token(s)
prompt eval duration: 264.066ms
prompt eval rate:     117.39 tokens/s
eval count:           70 token(s)
eval duration:        3.50308s
eval rate:            19.98 tokens/s
```

**Comment by BruceMacD** [comment]
Hi all, this may not be the case for everyone who experiences this issue, but if you encounter this the multimodal model may not be supported by your version of Ollama. Try updating if you're not on latest and running the model again.

**Comment by sam1am** [comment]
The issue is that the image is not being passed. It works in the interactive chat, but not in python, or when running the command from command line.

**Comment by Harrolee** [comment]
Updating Ollama resolved this behavior for me.

**Comment by mchiang0610** [comment]
Hi all, this should be fixed in the recent builds of ollama (0.1.28+). If you have older models, we might have changed the prompt templates to help prompt it better too (please try pulling the models again via `ollama pull` command) 

Sorry about this experience! Closing it for now, but please reach out if you are still seeing problems. Thanks!
